src/loaders/common/loader_common_names.py

```diff
@@ -83,4 +83,18 @@
 MASH_METADATA = 'mash_run_metadata.json'
 
 # The fatal error file created if a data file cannot be successfully processed
-FATAL_ERROR_FILE = "fatal_error.json"
\ No newline at end of file
+FATAL_ERROR_FILE = "fatal_error.json"
+
+# key name for sample file and prepared sample file in the metadata file for downloaded workspace objects
+SAMPLE_FILE_KEY = "sample_file"
+SAMPLE_PREPARED_KEY = "sample_prepared_file"
+
+# extension for source sample data file and prepared sample node data file for downloaded workspace objects
+SAMPLE_FILE_EXT = "sample"
+SAMPLE_PREPARED_EXT = "prepared.sample"
```

Comment:
Can you briefly mention in these comments that the standard key is the sample as retrieved and prepared is formatted for ingestion by the parser?

Reply:
👍

Reply:
:+1: